modules/ui.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
```

Comment:
- Please its not good practice to disable linters in your project. In microverse we encourage students to fix all linter errors and not disable them. Kindly fix this.